bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -359,4 +367,33 @@ public String getName() {
         return FormContainer.super.getName();
     }
 
+    @JsonIgnore
+    public String getFormDefinition() {
+        String result = null;
+        try {
```

Comment:
Can we move this to logic to a utility function

Reply:
Discussed with @rismehta, just move the logic into OperationUtils and reuse it here

Reply:
done